src/evict/evict_lru.c

```diff
@@ -914,6 +917,17 @@ __wt_evict_file_exclusive_on(WT_SESSION_IMPL *session)
     if (F_ISSET(session, WT_SESSION_PREFETCH_ENABLED))
```

Comment:
what about the session that takes exclusive access and doesn't enable the prefetch? we miss to clear the queue. Remove the `if` condition.